README.md

```diff
@@ -1,59 +1,52 @@
 The sample code in this directory demonstrates how to connect to CockroachDB with the [Rust Postgres driver](https://crates.io/crates/postgres).
 
-## Step 1. Specify the Rust Postgres driver as a dependency
+## Prerequisites
 
-Update your `Cargo.toml` file to specify a dependency on the Rust Postgres driver, as described in the [Rust Postgres driver](https://crates.io/crates/postgres).
+You must have rust and Cargo installed on your local machine, and a running CockroachDB cluster.
 
-Additionally, include the [OpenSSL bindings](https://crates.io/crates/postgres-openssl) and [Rust Postgres OpenSSL](https://crates.io/crates/postgres-openssl) crates as dependencies.
+## Step 1. Build the example
 
-## Step 2. Create the `maxroach` users and `bank` database
-
-Start the built-in SQL shell:
+In a terminal run the following command to download the dependencies and build the application.
 
 ~~~ shell
-$ cockroach sql --certs-dir=certs
-~~~
-
-In the SQL shell, issue the following statements to create the `maxroach` user and `bank` database:
-
-~~~ sql
-> CREATE USER IF NOT EXISTS maxroach;
+cd bank
+cargo build
```

Comment:
`cargo build` can be skipped because `cargo run` will automatically build the project as necessary.

Reply:
Done.